src/rest/SearchEndpoint.ts

```diff
@@ -632,11 +648,15 @@ export class SearchEndpoint implements ISearchEndpoint {
       }
     };
 
-    this.logger.info('Listing field values', request);
+    this.logger.info('Listing field batch values', request);
 
-    return this.performOneCall<IGroupByResult>(callParams, callOptions).then(data => {
-      this.logger.info('REST list field values successful', data.values, request);
-      return data.values;
+    interface IFieldValueBatchResponse {
```

Comment:
This interface should probably be exposed with the rest of the "restful" interface in `src/rest/`